airbyte-integrations/base-python/airbyte_protocol/airbyte_protocol/__init__.py

```diff
@@ -76,10 +70,34 @@ def __init__(self):
         pass
 
     # Iterator<AirbyteMessage>
-    def read(self, config_object, rendered_config_path, state=None) -> Generator[AirbyteMessage, None, None]:
+    def read(self, logger, config_container, catalog_path, state=None) -> Generator[AirbyteMessage, None, None]:
         raise Exception("Not Implemented")
 
 
 class Destination(Integration):
     def __init__(self):
         pass
+
+
+valid_log_types = ["FATAL", "ERROR", "WARN", "INFO", "DEBUG", "TRACE"]
```

Comment:
Don't you want to have a real object with real methods for the logging? Like `LOGGER` in java?